spec/rubocop/cop/rspec/repeated_description_spec.rb

```diff
@@ -117,4 +117,48 @@
       end
     RUBY
   end
+
+  it 'does not flag descriptions with different interpolated variables' do
+    expect_no_offenses(<<-RUBY)
+      describe 'doing x' do
+        it "does \#{x}" do
+        end
+
+        it "does \#{y}" do
+        end
+      end
+    RUBY
+  end
+
+  it 'registers offense for repeated description in same iterator' do
+    expect_offense(<<-RUBY)
+      describe 'doing x' do
+        %i[foo bar].each do |type|
+          it "does a thing \#{type}" do
+          ^^^^^^^^^^^^^^^^^^^^^^^^^ Don't repeat descriptions within an example group.
+          end
+
+          it "does a thing \#{type}" do
+          ^^^^^^^^^^^^^^^^^^^^^^^^^ Don't repeat descriptions within an example group.
+          end
```

Comment:
Do you mind to add an example of two separate iterators in the same example group? And maybe a balancing example showing that those having different docstrings are ignored. Not certain if it gets the reader an idea of how it works.

I already see people fooling the cop by naming variables differently, like `type1` and `type2` :D